core/build.sbt

```diff
@@ -443,7 +443,24 @@ def cloudflowModule(moduleID: String): Project = {
     .enablePlugins(AutomateHeaderPlugin)
 }
 
-lazy val commonSettings = Seq(
+// These settings are made active only when we use bintray for internal release
+// It is important that when we do final releases we need to invoke sbt as 
+// `sbt -Dsbt.sbtbintray=false`
```

Comment:
We could have this set to false by default. Testing should be optionally switched on, its like a dry-run flag.

Reply:
@skonto The logic is we will use `internalRelease` much more frequently than the actual one. Hence the plugin is on by default and we set it to false _only during an actual release_. See the other PR on `cloudflow-release`